avae/evaluate.py

```diff
@@ -31,12 +53,13 @@ def evaluate(datapath, lim, splt, batch_s, collect_meta, use_gpu):
     vae.to(device)
 
     # ########################## EVALUATE ################################
-    if collect_meta:
```

Comment:
Will be bringing this back due to issue raised by @marjanfamili

Reply:
Fixed in [1445299](https://github.com/alan-turing-institute/affinity-vae/pull/120/commits/1445299eeb021ff0597978e5e72fad30f5d2bac5)